src/main/app/src/Nexagon.js

```diff
@@ -0,0 +1,61 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {Hex, HexUtils} from 'react-hexgrid';
+
+
+class Nexagon extends Component {
```

Comment:
Is this correct or was it supposed to be named `NHexagon`?

Reply:
The whole naming can be refactored later